azurerm/internal/services/containers/kubernetes_nodepool.go

```diff
@@ -90,6 +90,14 @@ func SchemaDefaultNodePool() *schema.Schema {
 					ValidateFunc: validation.IntBetween(1, 100),
 				},
 
+				"node_labels": {
+					Type:     schema.TypeMap,
+					Optional: true,
+					Elem: &schema.Schema{
+						Type: schema.TypeString,
```

Comment:
I think we'll want to add validation here that prevents people from adding empty labels unless an empty label is valid?

Reply:
Empty label is valid.